README.md

```diff
@@ -37,20 +40,99 @@ By running `docker compose up -d` these steps take place:
 1. a [mongodb](./services/mongodb/) container is created with some initial data.
 2. the SciCat [backend v4](./services/backend/services/v4/) container is created and connected to (1).
 3. the SciCat [frontend](./services/frontend/) container is created and connected to (2).
-4. a reverse [proxy](./services/proxy) container is created and routes traffic to (2) and (3) through localhost subdomains, in the form: `http://${service}.localhost`. The frontend is available at simply `http://localhost`.
+4. a reverse [proxy](./services/proxy) container is created and routes traffic to (2) and (3) through localhost subdomains, in the form: `http://${service}.localhost`. The frontend is available at simply `http://localhost`. 
+5. Some services have additional endpoints that can be explored in SciCatLive which would follow `https://{service}.localhost/{prefix}`. For example, the backend API can be explored through a Swagger UI at `http://backend.localhost/explorer`.  For more information on the paths used by these routes see the original documentation for these services. 
```

Comment:
sorry, I've just noticed this should be `http://${service}.localhost/${prefix}` if we want to be consistent with point line before. I am happy to change point 4 if you like the line 5 notation better (the only other thing is to use http rather than https in point 5)

Reply:
Made that change